src/objects.js

```diff
@@ -1,41 +1,76 @@
 const createPerson = (name, age) => {
-  // your code here
+  // Create an object based on the name and age arguments
```

Comment:
As before all these comments can should be removed

Reply:
Will do